### PR TITLE
Fix failing tests due to nameval changes

### PR DESCRIPTION
**Pull request by daved**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-1680" title="DX-1680" target="_blank">BugDX-1680</a>  Test failure after merge to v38 (NameValue)
  </td></table>
  <br />

### CODE REVIEW

File: activestate.yaml [hunk 1]
```diff
@@ -262,7 +264,13 @@ scripts:
...
       export INTEGRATION_TEST_USERNAME=$secrets.project.INTEGRATION_TEST_USERNAME
       export INTEGRATION_TEST_PASSWORD=$secrets.project.INTEGRATION_TEST_PASSWORD
-      go test `go list ./... | grep "integration\|automation"` -v "${@:1}" -timeout 20m
+      go test `go list ./... | grep "${INTEGRATION_TEST_REGEX}"` -v "${@:1}" -timeout 20m
+  - name: integration-tests-build-only
+    language: bash
+    description: Builds integration tests and removes the .test artifacts.
+    value: |
+      go list ./... | grep "${INTEGRATION_TEST_REGEX}" | xargs --max-lines=1 go test -c
+      find . -type f -name "*.test" | grep "${INTEGRATION_TEST_REGEX}" | xargs rm
```
Comment (Naatan) [inline activestate.yaml:273]: What's the point of this? Doesn't seem related to the bug this PR is solving.

**Review by Naatan — CHANGES_REQUESTED** [review]

File: activestate.yaml (hunk 1 above)
Comment (daved) [reply activestate.yaml:273]: By building the tests, incorrect/outdated usage of types within the tests will be caught by the compiler.

**Review by Naatan — APPROVED** [review]
